app/controllers/ems_common.rb

```diff
@@ -1005,6 +1006,7 @@ def process_emss(emss, task)
           :target_class => model.to_s)
     elsif task == "destroy"
       model.where(:id => emss).order("lower(name)").each do |ems|
+        ems.disable_ems
```

Comment:
I dont think this should happen in the UI. 

But it seems that `model.destroy_queue(emss)` just queues ActiveRecord#destroy 
In our case we want something like an orchestrated destroy, but I'm not sure how to do that properly...

Just like @cben said "delay the actual destruction"